sbt-version-policy/src/sbt-test/sbt-version-policy/mima-semver-spec/test

```diff
@@ -0,0 +1,49 @@
+> v0-1-0/publishLocal
+> reload
+
+# OK to introduce a non-breaking change during 0.1.x
+> v0-1-1/versionPolicyCheck
+> v0-1-1/publishLocal
+> reload
+
+# OK to introduce a source incompatibility during 0.1.x
+> v0-1-2/versionPolicyCheck
+> v0-1-2/publishLocal
+> reload
+
+# OK to introduce binary incompatibilities
+# during 0.1.x
+> v0-1-3/versionPolicyCheck
+> reload
+
+# OK to introduce binary incompatibilities in 0.2.x
+> v0-2-0/versionPolicyCheck
+> v0-2-0/publishLocal
+> reload
+
+# New major version always succeeds
+> v1-0-0/versionPolicyCheck
+> v1-0-0/publishLocal
+> reload
+
+# OK to apply backward binary compatible changes
+> v1-0-1/versionPolicyCheck
+> v1-0-1/publishLocal
+> reload
+
+# Unable to apply source incompatible changes between
+# two patch versions
+-> v1-0-2/versionPolicyCheck
```

Comment:
I thought it would be the case only with `early-semver`?

Reply:
After `1.0.0`, I would think Early Semver and Semver Spec would behave exactly the same way.